python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1266,6 +1283,46 @@ def convertIntToFloat(self, exposure):
 
         return newexposure
 
+    def ditherCounts(self, exposure, detectorConfig, fallbackSeed=12345):
+        """Dither the counts in the exposure.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            The raw exposure to be dithered.
+        detectorConfig : `lsst.ip.isr.OverscanDetectorConfig`
+            Configuration for overscan/etc for this detector.
+        fallbackSeed : `int`, optional
+            Random seed to fall back to if exposure.getInfo().getId() is
+            not set.
+        """
+        if detectorConfig.integerDitherMode == "NONE":
+            # Nothing to do here.
+            return
+
+        # This ID is a unique combination of {exposure, detector} for a raw
+        # image as we have here.
+        seed = exposure.info.id
+        if seed is None:
+            seed = fallbackSeed
+            self.log.warning("No exposure ID found; using fallback random seed %d.", fallbackSeed)
+
+        rng = numpy.random.RandomState(seed=seed)
+
+        if detectorConfig.integerDitherMode == "POSITIVE":
```

Comment:
Do we expect these will differ between detectors?

Reply:
No, I guess not.  If you think it should be moved out of the overscan config to the regular config I can do that.

Reply:
I'm fine with it here for now.  We can simplify in the future if needed.